packages/app/src/systems/Overlay/components/OverlayDialog/OverlayDialog.tsx

```diff
@@ -1,19 +1,29 @@
 import { cssObj } from '@fuel-ui/css';
 import { Dialog } from '@fuel-ui/react';
 
+import { AccountSwitch } from '~/systems/Bridge/components/AccountSwitch';
 import { useOverlay } from '~/systems/Overlay';
 
-const OVERLAY_HEIGHT = 600;
-const OVERLAY_WIDTH = 600;
+const OVERLAY_HEIGHT = 100;
+const OVERLAY_WIDTH = 400;
 
 export function OverlayDialog() {
   const overlay = useOverlay();
 
   return (
     <Dialog isOpen={overlay.isDialogOpen}>
       <Dialog.Content css={styles.content}>
-        {/* example of how to use overlay \/. Should be removed after we add the first popup */}
-        {/* {overlay.is('accounts.list') && <Accounts />} */}
+        {/** TODO: remove the AccountSwitch component and replace with corresponding smart components once completed */}
+        {overlay.is('accounts.switch') && (
+          <AccountSwitch
+            accounts={[
+              'fuel17kx8dy6gvugrppnkvsezyyh2qxusq57mvk2hueaa9smer220knuslpsnuf',
```

Comment:
can we mock 3 accounts here to have a better view of how will behave with more accounts ?